File: ts/src/pull-with-push-override/run/run-foo-bar-override.ts
/*
 * Copyright © 2022 Paulo Villela. All rights reserved.
 * Use of this source code is governed by the MIT license
 * that can be found in the LICENSE file.
 */

import { fooSflCfgSrc, fooSfl } from "../fs/foo-sfl";
import { barBfCfgSrc } from "../fs/bar-bf";

fooSflCfgSrc.set(() => {
  return { x: "foo" }
})

barBfCfgSrc.set(() => {
  return { z: 99 }
})

fooSfl()

File: rs/hold/pullwithpushoverride/run/runFooBarOverride.kt
/*
 *  Copyright © 2022 Paulo Villela. All rights reserved.
 *  Use of this source code is governed by the MIT license
 *  that can be found in the LICENSE file.
 */

package pullwithpushoverride.run

import pullwithpushoverride.fs.*

fun main() {
	fooSflCfgSrc = { FooSflCfgInfo("foo") }

	barBfCfgSrc = { BarBfCfgInfo(99) }

	fooSfl()
}
